qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -449,7 +466,18 @@ bool HOCRPdfExporter::run(QString& filebasename) {
 	pdfSettings.uniformizeLineSpacing = ui.checkBoxUniformizeSpacing->isChecked();
 	pdfSettings.preserveSpaceWidth = ui.spinBoxPreserve->value();
 	pdfSettings.overlay = ui.comboBoxOutputMode->currentIndex() == 1;
-	pdfSettings.detectedFontScaling = ui.spinFontScaling->value() / 100.;
+	pdfSettings.detectedFontScaling = ui.spinFontScaling->value() / 100.0;
+	pdfSettings.paperSize = ui.comboBoxPaperSize->itemText(ui.comboBoxPaperSize->currentIndex());
+
+	if(pdfSettings.paperSize == "Custom") {
+		ui.lineEditPaperSizeWidth->setEnabled(true);
+		ui.lineEditPaperSizeHeight->setEnabled(true);
+	}
+	if(pdfSettings.paperSize != "Same as source") {
+		Size<int> outputPaperSize = PaperSizeProcessor::getSizeMM(pdfSettings.paperSize.toStdString());
+		ui.lineEditPaperSizeWidth->setText(QString::number(outputPaperSize.width));
+		ui.lineEditPaperSizeHeight->setText(QString::number(outputPaperSize.height));
+	}
```

Comment:
Shouldn't this be done in paperSizeChanged?

Reply:
Yep, it's just debug trash. Fixed.